packages/rpm/centos/mysql56-community-mroonga.spec.in

```diff
@@ -91,6 +91,12 @@ mv $RPM_BUILD_ROOT%{_datadir}/doc/mroonga/ mysql-mroonga-doc/
 rm -rf $RPM_BUILD_ROOT
 
 %post
+/sbin/service mysqld status
+if [ $? -ne 0 ] ; then
```

Comment:
I referred to this line, "evaluate $? after running command", 
https://github.com/mroonga/mroonga/blob/master/packages/rpm/centos/mysql56-community-mroonga.spec.in#L97

But the other hand, "evaluate directory in if condition" as you says,
https://github.com/mroonga/mroonga/blob/master/packages/rpm/centos/mysql56-community-mroonga.spec.in#L135

I'll fix it "evaluate directory in if condition" style :)